### DISCUSSION TITLE
渲染比较慢

### DISCUSSION THREAD

**Discussion by danchendoudou** [Happy-Coding-Clans/vue-easytable#465] [Q&A]
500行2000列的数据，渲染比较慢，有什么好的方法解决？

**Comment by huangshuwei** [comment]
已经开启了虚拟滚动吗，目前是支持行数据虚拟滚动，列数据虚拟滚动暂时还不支持

**Comment by danchendoudou** [comment]
虚拟滚动开启了，行数据没问题，我发现列数太多，渲染比较慢，只有几列的时候渲染会快很多




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Happy-Coding-Clans/vue-easytable"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月22日(星期二) 晚上7:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Happy-Coding-Clans/vue-easytable] 渲染比较慢 (Discussion #465)





 
已经开启了虚拟滚动吗，目前是支持行数据虚拟滚动，列数据虚拟滚动暂时还不支持
